.github/workflows/update-website.yaml

```diff
@@ -0,0 +1,75 @@
+name: Update Website
+on:
+  workflow_dispatch: {}
+  release:
+    types: [released]
+
+jobs:
+  release-images:
+    if: github.repository_owner == 'Apicurio'
+    runs-on: ubuntu-18.04 
+    steps:
+
+      - name: Apicurio Website Checkout
+        run: |
+          mkdir website
+          cd website
+          git init
+          git config --global user.name "apicurio-ci"
+          git config --global user.email "apicurio.ci@gmail.com"
+          git remote add origin "https://apicurio-ci:${{ secrets.ACCESS_TOKEN }}@github.com/Apicurio/apicurio.github.io.git"
+          git fetch
+          git checkout master
+          git branch --set-upstream-to=origin/master
+          git pull
+
+      - name: Apicurio Playbook Checkout
+        run: |
+          mkdir docs-playbook
+          cd docs-playbook
+          git init
+          git config --global user.name "apicurio-ci"
+          git config --global user.email "apicurio.ci@gmail.com"
+          git remote add origin "https://apicurio-ci:${{ secrets.ACCESS_TOKEN }}@github.com/Apicurio/apicurio-docs-playbook.git"
+          git fetch
+          git checkout master
+          git branch --set-upstream-to=origin/master
+          git pull
+
+      - name: Generate Project Documentation for Website
+        run: |
+          echo "********** Building Registry Documentation using Antora **********"
+          cd docs-playbook
+          docker pull apicurio/apicurio-docs-builder:latest
+          docker run -v $(pwd):/apicurio-docs-playbook apicurio/apicurio-docs-builder:latest
+          rm -rf ../website/registry/docs/*
+          cp -rf target/dist/* ../website/registry/docs/.
+
+      - name: Fetch required Details
+        run: echo "RELEASE_VERSION=$(curl https://api.github.com/repos/apicurio/apicurio-registry/releases/latest | jq '.name' | sed 's/"//g')" >> $GITHUB_ENV
+
+      - name: Updating Project Website for Release ${{ env.RELEASE_VERSION }}
+        run: |
+          cd website
+          echo "********** Replacing Existing Latest Release JSON File **********"
+          cd _data/registry
+          rm latestRelease.json
+          touch latestRelease.json && curl https://api.github.com/repos/apicurio/apicurio-registry/releases/latest > latestRelease.json
+          
+          echo "********** Saving a copy in the archieve **********"
+          cp latestRelease.json releases/${RELEASE_VERSION}.json
```

Comment:
Should this be `${{ env.RELEASE_VERSION }}` ?

Reply:
Oh nevermind, I see line 49 now.  But still, I'm worried about this line.  If we're using it as a linux ENV var, then maybe it should be this: 
`cp latestRelease.json releases/$RELEASE_VERSION.json`

I can't remember if it's OK to have brackets in there around the variable name.

Reply:
But now I see we use the `${{ env.RELEASE_VERSION }}` syntax later (line 66).  What do you think?  :)

Reply:
> Oh nevermind, I see line 49 now. But still, I'm worried about this line. If we're using it as a linux ENV var, then maybe it should be this:
> `cp latestRelease.json releases/$RELEASE_VERSION.json`
> 
> I can't remember if it's OK to have brackets in there around the variable name.

Both ${RELEASE_VERSION} and $RELEASE_VERSION are acceptable in linux.

Reply:
> But now I see we use the `${{ env.RELEASE_VERSION }}` syntax later (line 66). What do you think? :)

I am setting RELEASE_VERSION as environment variable at line 49. Every environment variable becomes available in the `env` context.  Now `env` is an object and I am using ${{ env.RELEASE_VERSION }}` to access the field `RELEASE_VERSION` from the `env` context. 

In short, ${RELEASE_VERSION} and $RELEASE_VERSION are Linux syntax. But what if we are using a windows runner, we can't use the Linux syntax. thus we the Jinja2 template i.e. ${{ env.RELEASE_VERSION }}

Reply:
No problem.  Should we use `${{ env.XXX }}` syntax on line 60 as well then?